OSDOCS#13430: Update the z-stream RN for 4.15.46 (#89262)
openshift/openshift-docs: release_notes/ocp-4-15-release-notes.adoc

====

// 4.15.46
[id="ocp-4-15-46_{context}"]
=== RHSA-2025:1711 - {product-title} 4.15.46 bug fix and security update

Issued: 27 February 2025

{product-title} release 4.15.46, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2025:1711[RHSA-2025:1711] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHSA-2025:1713[RHSA-2025:1713] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.15.46 --pullspecs
----

[id="ocp-4-15-46-bug-fixes_{context}"]
==== Bug fixes

* Previously, if you tried to rerun a resolver-based `PipelineRun` from the {product-title} console, the "Invalid PipelineRun configuration, unable to start Pipeline" UI message was displayed. With this release, you can rerun a resolver-based `PipelineRun` with no problem. (link:https://issues.redhat.com/browse/OCPBUGS-48593[*OCPBUGS-48593*])

> * Previously, a bug caused requests to update the `deploymentconfigs/scale` sub resource to fail when a matching admission webhook was configured. With this release, a fix allows the update to continue without an error. (link:https://issues.redhat.com/browse/OCPBUGS-47766[*OCPBUGS-47766*])

Reviewer: ```suggestion
* Previously, a bug caused requests to update the `deploymentconfigs/scale` sub resource to fail when a matching admission webhook was configured. With this release, you can update to continue without an error. (link:https://issues.redhat.com/browse/OCPBUGS-47766[*OCPBUGS-47766*])
```

Avoid using "allow": https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a

> Avoid stating that inanimate objects grant abilities to people, as in “the product allows you to…”. Whenever possible, use a direct, user-focused alternative such as “you can use the product to…” or “with this product, users can…”.

> * Previously, the installation program did not validate the MTU for custom networks on Red Hat OpenStack platforms, which led to an installation failure when the MTU was too small. For IPv6, the minimum MTU is 1280 and 100 for OVN-Kubernetes. With this release, the installation program validates the MTU of Red Hat OpenStack custom networks. (link:https://issues.redhat.com/browse/OCPBUGS-41815[*OCPBUGS-41815*])

Reviewer: ```suggestion
* Previously, the installation program did not validate the maximum transmission unit (MTU) for custom networks on Red{nbsp}Hat OpenStack platforms, which led to an installation failure when the MTU was too small. For IPv6, the minimum MTU is 1280 and 100 for OVN-Kubernetes. With this release, the installation program validates the MTU of Red{nbsp}Hat OpenStack custom networks. (link:https://issues.redhat.com/browse/OCPBUGS-41815[*OCPBUGS-41815*])
```

https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces

> Use a non-breaking space ({nbsp}) between the words "Red" and "Hat". The non-breaking space prevents an automatic line break from separating the two words onto two lines.
